packages/react/src/ui/indicator.tsx

```diff
@@ -21,11 +23,12 @@ export const Indicator = forwardRef<HTMLDivElement, IndicatorProps>(
           >
             {label}
           </p>
-          {icon && (
+          {icon && !emoji && (
             <span className={cn("flex", color)}>
               <Icon icon={icon} />
             </span>
           )}
+          {emoji && !icon && <EmojiImage emoji={emoji} size="md" />}
```

Comment:
if i pass both emoji and icon, I get nothing. Let's make impossible states impossible. For that, we have to modify the prop:
```
type IndicatorProps = {
  content: string
  label: string
  color?: string
  emoji: string
} | {
  content: string
  label: string
  color?: string
  icon: IconType
}
```

We explicitly allow only `icon` or `emoji` prop, making the situation when both things are passed impossible.